src/randoop/main/GenBranchDir.java

```diff
@@ -181,9 +182,13 @@ public static void main(String[] args) {
     }
     Set<CoverageAtom> allBranches = new LinkedHashSet<CoverageAtom>();
     for (String className : covClassNames) {
-      Class<?> cls = Reflection.classForName(className);
+      try {
+      Class<?> cls = TypeNames.recognizeType(className);
       covClasses.add(cls);
       allBranches.addAll(Coverage.getBranches(cls));
+      } catch (ClassNotFoundException e) {
+        //ignore
```

Comment:
Every time I see a catch clause ignoring a thrown exception, it gives me the heebie jeebies.  Can you please eliminate all uses of that undesirable code pattern from Randoop?